client/state/invites/actions.js

```diff
@@ -187,8 +187,8 @@ export function createAccount( userData, invite ) {
 		result
 			.then( () => {
 				recordTracksEvent( 'calypso_invite_account_created', {
-					is_p2_site: get( invite, 'site.is_wpforteams_site', false ),
-					inviter_blog_id: get( invite, 'site.ID', false ),
+					is_p2_site: invite?.site?.is_wpforteams_site ?? false,
+					inviter_blog_id: invite?.site?.ID ?? false,
```

Comment:
We're using `invite.sendTo` above so it's already considered an object. Therefore we should be able to remove the first optional chaining:

```suggestion
					is_p2_site: invite.site?.is_wpforteams_site ?? false,
					inviter_blog_id: invite.site?.ID ?? false,
```